functional/README.md

```diff
@@ -0,0 +1,139 @@
+# Functional Test Framework
+`go test`-like functional testing framework.
+
+## Why use `functional`?
+`functional` is used when you want to rapidly develop code in the format of a
+unit test or benchmark test using existing testing tools, but run it outside of
+the `go test` environment.  This is handy for use cases needing data inspection
+and having low tolerance for errors.
+
+`go test` doesn't support running tests programmatically from compiled code; it
+requires the source code, which won't be available in production.
```

Comment:
Is it still compatible with `-ldflags "-s -w"` flags? Or does it require all these DWARF debugging information and symbol table?

Reply:
Cool.